ui/app/clinical/common/views/visitTabPrint.html

```diff
@@ -19,6 +19,6 @@ <h1>
         </div>
         <div ng-include="currentVisitUrl"></div>
     </div>
-    <div class="provider-info">{{provider[0].name}}</div>
+    <div ng-if="visitTabConfig.getPrintConfigForCurrentTab().hideProviderNane" class="provider-info">{{provider[0].name}}</div>
```

Comment:
variable name has a typo: `hideProviderName`.

Reply:
Also, if this variable is not present, then it will be "false" in which case by default the provider name will get hidden. Would prefer that by default this is shown (if variable is not defined). Can u invert the logic?